lib/cisco_node_utils/cmd_ref/acl_v4.yaml

```diff
@@ -0,0 +1,25 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^ip access-list <acl_name>$/'
```

Comment:
@chrisvanheuveln As we discussed it would work because in the constructor we are passing `ipv4` or `ipv6` so in the `ipv4` case we simply use `ip` as the `@get_args` value for `:afi`